corehq/form_processor/backends/couch/dbaccessors.py

```diff
@@ -102,6 +103,10 @@ def get_attachment_content(form_id, attachment_id):
     def save_new_form(form):
         form.save()
 
+    @staticmethod
+    def update_form(form):
+        form.save()
```

Comment:
Is this (and the superclass version) used anywhere?

Reply:
Nope! This is a remnant from something I tried earlier that isn't relevant anymore. 
Removed them both here:
https://github.com/dimagi/commcare-hq/pull/19979/commits/3785fe4ab63bca7fa19dd033723a144151077093